### ISSUE TITLE
Data download broken

### ISSUE THREAD

**Issue by tomkooij** [HiSPARC/publicdb#222]
**all** data downloads seem to be broken (empty .tsv returned)

`/var/log/uwsgi.log`
```
[pid: 23304|app: 0|req: 1026/74205] 145.102.135.182 () {40 vars in 1188 bytes} [Mon Apr  9 09:08:22 2018] GET /raw_data/download/?data_type=events&station_events=182&station_weather=182&lightning_type=4&station_singles=182&start=2018-04-07&end=2018-04-08&download=on => generated 0 bytes in 20 msecs (HTTP/1.1 302) 3 headers in 180 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "../publicdb/raw_data/views.py", line 270, in generate_events_as_tsv
    yield t.render(c)
  File "/srv/publicdb/publicdb_env/lib/python2.7/site-packages/django/template/backends/django.py", line 64, in render
    context = make_context(context, request, autoescape=self.backend.engine.autoescape)
  File "/srv/publicdb/publicdb_env/lib/python2.7/site-packages/django/template/context.py", line 287, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than Context.
[pid: 23306|app: 0|req: 16303/74206] 145.102.135.182 () {40 vars in 1050 bytes} [Mon Apr  9 09:08:22 2018] GET /data/4/events/?download=True&start=2018-04-07+00%3A00%3A00&end=2018-04-08+00%3A00%3A00 => generated 0 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 160 bytes (0 switches on core 0)

```

I would like to revert `pique` to a last known good state, but since django was updated, I'll probably break other things, so let's just try to fix things.

@153957 any thoughts?

**Comment by 153957** [comment]
You can try to revert Django to 1.10.x, that might be the easiest fix.

**Comment by tomkooij** [comment]
I revert to `Django=1.10` to fix it. (Thanks!)

I'm leaving this open while this gets fixed.

**Comment by 153957** [comment]
Simply remove the `Context()` wrappers around the dictionaries in the raw data views: https://github.com/HiSPARC/publicdb/blob/master/publicdb/raw_data/views.py

**Comment by tomkooij** [comment]
Fixed in dc19a6ffa3f8b0d4c2a3c32287bf99e53608f04
